tests/test_styles.py

```diff
@@ -1,9 +1,9 @@
 import os
 import sys
-import pytest
```

Comment:
No reason to reorder this right?

Reply:
There is: [PEP 8 suggests](https://www.python.org/dev/peps/pep-0008/#imports) an ordering of standard library, 3rd party library, own libraries/helper